fdbserver/workloads/BackupAndParallelRestoreCorrectness.actor.cpp

```diff
@@ -484,7 +486,8 @@ struct BackupAndParallelRestoreCorrectnessWorkload : TestWorkload {
 					                                       deterministicRandom()->randomInt(0, 100),
 					                                       self->backupTag.toString(),
 					                                       self->backupRanges,
-					                                       SERVER_KNOBS->ENABLE_ENCRYPTION,
+					                                       SERVER_KNOBS->ENABLE_ENCRYPTION &&
+					                                           configuration.tenantMode != TenantMode::OPTIONAL_TENANT,
```

Comment:
I would prefer add an `disableEncryptionWithOptionalTenantMode` flag and use it in test specs.

Reply:
But each test runs with a random probability of having optional/required/disabled tenant mode so i can't really determine this at the test spec level

Reply:
Unless you mean guarding the use with this flag? There are quite a few workloads so I don't know how feasible/clean this would be

Reply:
i see i think its probably better for now to enforce at the workload level rather than the test file level otherwise each time a new toml file gets created someone will need to explicitly use this parameter otherwise the test might fail